### PR TITLE
Refactor, bugfixes, and verification script

### PR DESCRIPTION
**Pull request by ErikBjare**

### CODE REVIEW

File: aw_watcher_afk/afk.py [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
 # TODO: Move to argparse
 settings = {
-    "timeout": 180,
-    "update_interval": 30,
+    "timeout": 30,
```
Comment (ErikBjare) [inline aw_watcher_afk/afk.py]: I need to change this back

File: tests/test_continuous_events.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from datetime import timedelta
+
+from aw_client import ActivityWatchClient
+
+client = ActivityWatchClient("aw-watcher-afk", testing=True)
+print(client.get_buckets())
+
+events = client.get_events("aw-watcher-afk-testing_erb-laptop-ubuntu")
```
Comment (ErikBjare) [inline tests/test_continuous_events.py]: This needs to be generalized, or even better made into an argument to the script

**Comment by ErikBjare** [comment]
This is at least better than the last version, I'm working on some further improvements to make the code clearer but I'm merging this now since I think it's pretty low-risk and somewhat well tested (I've had it running for >48h with few issues according to the validation script).

**Comment by johan-bjareholt** [comment]
Nice work, will try it now! :)

**Pull request merged**